decidim-core/config/i18n-tasks.yml

```diff
@@ -85,6 +85,7 @@ search:
 ## Do not consider these keys missing:
 ignore_missing:
 - '{devise,simple_form}.*'
+- errors.messages.taken
```

Comment:
This belongs to rails, and i18n-tasks complains about it as it isn't inside our project.